ee-feature-pack/galleon-content/src/main/resources/feature_groups/standalone-core.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feature-group-spec name="standalone-core" xmlns="urn:jboss:galleon:feature-group:1.0">
+
+    <feature spec="server-root">
```

Comment:
This feature-group is not included in the ee-core-profile-server layer. Is it really needed?

Reply:
TBH I don't know. I just copied what `standalone-full.xml` does and tried to modify what I thought would not be needed.

Reply:
Where is `server-root` defined? I can't find anything in core or full where it's actually defined.

Reply:
https://github.com/wildfly/wildfly-core/blob/main/core-feature-pack/galleon-common/src/main/resources/layers/standalone/base-server/layer-spec.xml
that contains standalone-base that contains it. All layers depend on base-server layer, so you will have it provisioned.

Reply:
Perfect, thanks!